e2e/Dockerfile.hello-microshift

```diff
@@ -0,0 +1,19 @@
+FROM busybox:1.36
```

Comment:
Can we pick an image from quay? Just have in mind we need both amd and arm

Reply:
Created a busybox mirror `quay.io/microshift/busybox:1.36` with [skopeo-copy](https://github.com/containers/skopeo/blob/main/docs/skopeo-copy.1.md#examples). 

Busybox supports both `amd` and `arm`

```
podman manifest inspect quay.io/microshift/busybox:1.36
```

```json
{
    "schemaVersion": 2,
    "mediaType": "application/vnd.docker.distribution.manifest.list.v2+json",
    "manifests": [
        ...
        ...
        {
            "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
            "size": 528,
            "digest": "sha256:acaddd9ed544f7baf3373064064a51250b14cfe3ec604d65765a53da5958e5f5",
            "platform": {
                "architecture": "amd64",
                "os": "linux"
            }
        },
        ...
        ...
        {
            "mediaType": "application/vnd.docker.distribution.manifest.v2+json",
            "size": 528,
            "digest": "sha256:ba7000206594c2d72c3ab550453004c0dc50961157e5ebd2fb8ea1890099d02d",
            "platform": {
                "architecture": "arm64",
                "os": "linux",
                "variant": "v8"
            }
        }
    ]
}

```

However, I am unsure whether Podman `Build trigger` supports muti-arch build or not. If not, may be we need to think of a CI job for this.